src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -255,17 +264,23 @@ class PerfherderGraphContainer extends Component {
 
 PerfherderGraphContainer.propTypes = {
   classes: PropTypes.shape({}).isRequired,
-  series: PropTypes.arrayOf(
-    PropTypes.shape({
-      label: PropTypes.string.isRequired,
-      filter: PropTypes.shape({}).isRequired,
-      options: PropTypes.shape({
-        includeSubtests: PropTypes.bool,
-      }),
-    })
-  ),
+  series: PropTypes.oneOfType([
```

Comment:
@helfi92  What duplication did you see?

Reply:
I don't see a duplication right now but for future views that may take a serie as a prop, it would be nice to import that prop from a `prop-types` files instead of having to re-define it. I assumed a serie is a general data structure that may be consumed by future components.